db/seed.js

```diff
@@ -7,17 +7,20 @@ var rnd = n => Array.isArray(n) ? n[ rnd(n.length) - 1] : Math.floor( Math.rando
`,`Card Game`,`Action RPG`,`Resource Management`,`Dark Fantasy`,`3D Platformer`,`Base Building`,`Romance`,`Post-apocalyptic`,`Turn-Based Tactics`,`Logic`,`Drama`,`PvP`,`Stealth`,`Dating Sim`,`Surreal`,`Futuristic`,`Hand-drawn`,`Animation & Modeling`,`Abstract`,`Isometric`,`Medieval`,`Masterpiece`,`Economy`,`Board Game`,`Dark Humor`,`Character Customization`,`Experimental`,`Mouse only`,`Twin Stick Shooter`,`Driving`,`Metroidvania`,`Addictive`,`Interactive Fiction`,`Cyberpunk`,`Magic`,`Perma Death`,`City Builder`,`Robots`,`Choose Your Own Adventure`,`Soundtrack`,`Match 3`,`Cartoon`,`Third-Person Shooter`,`Military`,`Flight`,`Beautiful`,`3D`,`Score Attack`,`Video Production`,`Clicker`,`Aliens`,`Arena Shooter`,`Audio Production`,`Detective`,`Level Editor`,`1980s`,`World War II`,`Destruction`,`Psychological`,`Blood`,`2D Fighter`,`Rhythm`,`LGBTQ+`,`Thriller`,`Moddable`,`2.5D`,`Lovecraftian`,`Game Development`,`Time Management`,`Real-Time`,`GameMaker`,`Space Sim`,`Episodic`,`Dark Comedy`,`Team-Based`,`Alternate History`,`Runner`,`Real-Time with Pause`,`Parkour`,`Loot`,`Strategy RPG`,`Demons`,`Narration`,`Software Training`,`Voxel`,`Science`,`Steampunk`,`Grand Strategy`,`Battle Royale`,`Hex Grid`,`Wargame`,`Tanks`,`Split Screen`,`Tactical RPG`,`Pirates`,`Dystopian`,`Kickstarter`,`Political`,`Real Time Tactics`,`PvE`,`4X`,`Mechs`,`Crime`,`Linear`,`Dungeons & Dragons`,`Otome`,`Dinosaurs`,`Nature`,`Time Attack`,`Psychedelic`,`Souls-like`,`Documentary`,`Ninja`,`Dragons`,`Tutorial`,`Touch-Friendly`,`Cult Classic`,`Illuminati`,`Swordplay`,`Word Game`,`Politics`,`Trains`,`Western`,`Grid-Based Movement`,`Nonlinear`,`Agriculture`,`Naval`,`eSports`,`Lore-Rich`,`NSFW`,`Faith`,`Comic Book`,`Hacking`,`Experience`,`MMORPG`,`God Game`,`MOBA`,`Character Action Game`,`Crowdfunded`,`Emotional`,`Modern`,`CRPG`,`Underwater`,`Remake`,`Philisophical`,`Programming`,`Minigames`,`Football`,`Photo Editing`,`Superhero`,`Silent Protagonist`,`Games Workshop`,`Cats`,`Satire`,`Cold War`,`Hunting`,`Co-op Campaign`,`FMV`,`Mythology`,`Soccer`,`Noir`,`Cinematic`,`Time Manipulation`,`Music-Based Procedural Generation`,`Bullet Time`,`Parody`,`360 Video`,`Fishing`,`Gothic`,`Capitalism`,`Supernatural`,`Artificial Intelligence`,`Gun Customization`,`Trading Card Game`,`Investigation`,`Martial Arts`,`Time Travel`,`Mystery Dungeon`,`Offroad`,`America`,`Trading`,`Inventory Management`,`Vampire`,`Sokoban`,`Chess`,`Mars`,`Mining`,`6DOF`,`Typing`,`Gaming`,`Solitaire`,`Assassin`,`Villain Protagonist`,`Spectacle fighter`,`Golf`,`Warhammer 40K`,`Conversation`,`Dog`,`Sailing`,`Epic`,`Basketball`,`Sniper`,`World War I`,`Motorbike`,`On-Rails Shooter`,`Quick-Time Events`,`Underground`,`Diplomacy`,`Horses`,`Sequel`,`Dynamic Narration`,`Asynchronous Multiplayer`,`Unforgiving`,`Conspiracy`,`Bikes`,`Mini Golf`,`Pinball`,`Submarine`,`Mod`,`Class-Based`,`Rome`,`LEGO`,`Gambling`,`Heist`,`Intentionally Awkward Controls`,`3D Vision`,`Baseball`,`Tennis`,`Based On A Novel`,`Wrestling`,`Immersive Sim`,`Transhumanism`];
```

Comment:
Any reason for not camel casing?

Reply:
Not really - I had just been using both indiscriminately and wanted to pick one. I don't really have an attachment to either - I think I'd just used more underscores than camelCases, so I just decided to switch to that. If camelCase is a better option, I'd be happy to switch them up.